src/providers/dataProvider/index.ts

```diff
@@ -129,15 +129,30 @@ export const getDataProvider = async (
     {
       resource: constants.R_USERS,
       afterDelete: async (record: DeleteResult<User>) => {
-        await audit(AuditType.DELETE_USER, `User deleted (${record.data.id})`)
+        await audit({
+          type: AuditType.DELETE_USER,
+          activityDetail: `User deleted (${record.data.id})`,
+          resource: constants.R_USERS,
+          item: null
+        })
         return record
       },
       afterCreate: async (record: CreateResult<User>) => {
-        await audit(AuditType.CREATE_USER, `User created (${record.data.id})`)
+        await audit({
+          type: AuditType.CREATE_USER,
+          activityDetail: `User created (${record.data.id})`,
+          resource: constants.R_USERS,
+          item: null
```

Comment:
aah, this should be `id`, not `item`, and it should take `record.data.id` - we can `eventually` list all audit events for this user record.